webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -1013,28 +1011,42 @@ <h3>Download Raw Crash Data and Minidumps</h3>
                   See <a href="{{ url('documentation:protected_data_access') }}">Protected data policy</a> for details.
                 {% endif %}
               </p>
-              <ul>
-                {% for name, url in raw_dump_urls %}
+              <ul class="list-inside list-disc">
+                {% for name, url in raw_data_urls %}
                   <li>{{ name }}: <a href="{{ url }}">{{ url }}</a></li>
                 {% endfor %}
-                {% set unredacted_url = url('api:model_wrapper', model_name='UnredactedCrash') + '?' + make_query_string(crash_id=report.uuid) %}
-                <li>processed crash: <a href="{{ unredacted_url }}" target="_blank">{{ unredacted_url }}</a>
               </ul>
             {% elif request.user and request.user.is_active %}
               <p>
                 You do not have access to protected data.
-                You need to be logged in and have access to protected data to see this.
+                You need to be logged in and have access to protected data to see links to crash report data.
                 See <a href="{{ url('documentation:protected_data_access') }}">protected data access documentation</a>
                 for more information.
               </p>
             {% else %}
               <p>
                 You are not logged in.
-                You need to be logged in and have access to protected data to see this.
+                You need to be logged in and have access to protected data to see links to crash report data.
                 See <a href="{{ url('documentation:protected_data_access') }}">protected data access documentation</a>
                 for more information.
               </p>
             {% endif %}
+
+            <h3>minidump-stackwalk output</h3>
+            <div>
+              {% if request.user.has_perm('crashstats.view_rawdump') %}
+                {{ protected_warning(your_crash) }}
+                {% if not your_crash %}
+                  minidump-stackwalk output contains sensitive and exploitability information.
+                {% endif %}
+              {% else %}
+                You do not have access to protected data so the minidump-stackwalk output is redacted.
+                See <a href="{{ url('documentation:protected_data_access') }}">protected data access documentation</a>
+                for more information.
+              {% endif %}
+            </div>
+            <div id="minidump-stackwalk-json" data-minidumpstackwalk="{{ raw_stackwalker_output }}"></div>
```

Comment:
This displays the minidump-stackwalk output the same way we do the Telemetry environment output. It's got some nice features that make it easier to see a HUGE PILE O' JSON.